src/components/banner/banner.js

```diff
@@ -1,32 +1,59 @@
 import React from 'react';
-import banner from './banner.svg';
 import PropTypes from 'prop-types';
-
 import { withStyles } from '@material-ui/core/styles';
 
+import Wrenches from './wrenches';
+
 const styles = theme => ({
 	container: {
-		position: 'relative'
+		backgroundColor: '#352A3A'
+	},
+	content: {
+		display: 'flex',
+		position: 'relative',
+		alignItems: 'center',
+		justifyContent: 'left',
+		maxWidth: '800px',
+		height: '400px',
+		margin: 'auto',
+		[theme.breakpoints.down('xs')]: {
+			maxWidth: '370px',
+			maxHeight: '250px'
+		}
 	},
 	title: {
 		position: 'absolute',
-		right: theme.spacing.unit * 9.375,
-		top: theme.spacing.unit * 15.625,
+		right: '0px',
 		textAlign: 'right',
 		color: 'white',
 		fontFamily: ['Poppins', 'sans-serif'],
-		fontWeight: 'bold'
+		fontWeight: 'bold',
+		minWidth: 'max-content',
```

Comment:
What does this mean?

Reply:
This just auto-sizes the width of the divider so that it expands to the space that the title takes up, but doesn't have extra space around it.